ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -424,6 +428,7 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
     @objc public class func preferredWidth(title: String,
                                            subtitle: String = "",
                                            footer: String = "",
+                                           hasLeadingDot: Bool = false,
```

Comment:
Is this going to be a breaking change?

Reply:
Good catch, yes it is. I'll put the label on!